### DISCUSSION TITLE
Any way to simplify speed settings for OrcaSlicer?

### DISCUSSION THREAD

**Discussion by eSonOfAnder** [OrcaSlicer/OrcaSlicer#4149] [General]
I am coming mostly from Cura where there is a sort of "master" print speed that the various other print speeds are percentages of. There does not appear to be this concept within OrcaSlicer which makes updating print speeds a bit of a pain, because you're stuck having to change each value individually when you want to make any updates. Still learning my way around the new slicer, but this one jumped out at me as being a bit non-intuitive and unnecessarily fiddly.

**Comment by ProtomakerSprint** [comment]
I load in a large model that I previously used MeshMixer to Thinking.

OrcaSlicer suggested that I "simplify" because it said it would take a long time to print; previously it has taken about 5 hours to print

After using the simplify  option, it took about 3 1/3 hours, expected still printing it, I _think it_ makes lower infill and reduce walls and speed up, so weaking the part; but it is only a test fit print anyway for me

In OrcaSlicer I expect there is an option to do this on prints somewhere; don't know if this is so.

**Remember you make get worse quality and weaker parts**

**Reply by eSonOfAnder** [reply]
As far as I know, there is not, but that is also not what I am talking about here. What I am requesting is something akin to how Cura allows you to set one setting called "Print Speed", which is effectively the max print speed, and all of the individual print speeds are based off of that value by some percentage.

e.g. I set a Print Speed of 100, that automatically sets all of the infill speeds to 100, the outer walls to 50 (50% of 100), travel speed to 125, first layer to 40, etc. 

These ratios tend to be pretty static, so rather than goong through and changing every single one if you simply want to adjust the overall print speed is cumbersome and awkward at best. Allow for adjusting those ratios to fine tune the individual settings, but I would expect 95%+ of the time these aren't ever going to change, just the overall print speeds.

**Reply by ProtomakerSprint** [reply]
Orcaslicer has a magnifying glass which will all a search on: speed;

Cura was indeed easier
